src/main/java/org/apache/accumulo/testing/continuous/ContinuousIngest.java

```diff
@@ -177,10 +183,12 @@ public static void main(String[] args) throws Exception {
         for (int depth = 1; depth < maxDepth; depth++) {
           for (int index = 0; index < flushInterval; index++) {
             long rowLong = genLong(rowMin, rowMax, r);
-            byte[] prevRow = genRow(prevRows[index]);
-            prevRows[index] = rowLong;
-            Mutation m = genMutation(rowLong, r.nextInt(maxColF), r.nextInt(maxColQ), cv,
-                ingestInstanceId, count, prevRow, checksum);
+            byte[] prevRow = genRow(nodeMap[depth - 1][index].row);
+            int cfInt = r.nextInt(maxColF);
+            int cqInt = r.nextInt(maxColQ);
+            nodeMap[depth][index] = new MutationInfo(rowLong, cfInt, cqInt);
```

Comment:
We may be able to avoid creating these MutationInfo objects when we are not deleting.  Would need to move the creation of the delete var to before this loop.  Less object creation may speed things up a bit when we are not deleting.

```suggestion
            if(delete || depth == maxDepth - 1) {
               nodeMap[depth][index] = new MutationInfo(rowLong, cfInt, cqInt);
            }
```

Reply:
Don't we need these values in order to create the `byte[] prevRow` regardless of whether we are deleting or not?

Reply:
Yeah we do need them for that, I was only looking at the later code.